usagov/vote-gov-drupal
# VOTE-823 VOTE-824 remove publication date module

Update bixaluswds theme to be managed by composer
Remove unused stable theme



## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-823
https://cm-jira.usa.gov/browse/VOTE-824

## Description

Remove unused modules and update bixaluswds theme

## Deployment and testing

### Pre-deploy
*BEFORE CHECKING OUT THIS BRANCH!!!*
1. checkout `dev` branch and import latest db
2. run `lando retune`
3. log into your local
4. visit these form display settings pages and move "published on" field to "disabled" category
http://vote-gov.lndo.site/admin/structure/types/manage/page/form-display
http://vote-gov.lndo.site/admin/structure/types/manage/landing/form-display
http://vote-gov.lndo.site/admin/structure/types/manage/nvrf_page/form-display
http://vote-gov.lndo.site/admin/structure/types/manage/state_territory/form-display
http://vote-gov.lndo.site/admin/structure/types/manage/voter_guide/form-display
5. visit http://vote-gov.lndo.site/admin/modules/uninstall and uninstall Publication Date module
6. checkout `feature/VOTE-823-pubdate-remove` branch

### Post-deploy

1. run `lando retune`

### QA/Test

1. visit http://vote-gov.lndo.site/node/3/edit and check to make sure there is no publication date field in the sidebar
2. visit http://vote-gov.lndo.site/about-us and make sure the theme is still loading and there are no console log issues

## Checklist for the Developer




## Checklist for the Peer Reviewers
